mediawiki/README.md

```diff
@@ -147,6 +147,8 @@ volumes:
 
 Run `docker stack deploy -c stack.yml mediawiki` (or `docker-compose -f stack.yml up`), wait for it to initialize completely, and visit `http://swarm-ip:8080`, `http://localhost:8080`, or `http://host-ip:8080` (as appropriate).
 
+When specifying the database host as part of the wiki setup process, make sure to use `database://localhost`.
```

Comment:
Hmm, `localhost` wouldn't be correct though, because that would imply the container `mediawiki` is running in -- rather, that should be something like `database://db`, right?  (This would also need to be part of `content.md` as `README.md` is generated, so it might be even better to include it as a code comment inside `stack.yml`)

Also cc `mediawiki` image maintainers: @legoktm @addshore @christian-heusel

Reply:
`localhost` would be correct though because there's are separate `mediawiki` and `database` services. For the `mediawiki` container to connect to the database it needs to use the `localhost` of the `database` container which the MySQL database is exposing itself to.

Reply:
`localhost` might work but it's not ideal to leave the compose environment and the name of the service should be used instead (`database` in this case).